normandy/recipes/tests/test_exports.py

```diff
@@ -25,35 +25,24 @@ def rs_urls(self, rs_settings):
             "workspace": {"capabilities": {}, "baseline": {}},
```

Comment:
Since we don't have to vary based on capabilities vs baseline, we should remove the second layer of this hierarchy. So instead of `rs_urls["workspace"]["capabilities"]["records"]` we'd have `rs_urls["workspace"]["records"]`.